src/plugins/vis_builder/public/application/utils/get_top_nav_config.tsx

```diff
@@ -95,7 +95,7 @@ export const getTopNavConfig = (
               dispatch,
               services
             )}
-            objectType={'wizard'}
+            objectType={VIS_BUILDER_TYPE}
```

Comment:
The existing naming here is a bit confusing based on our UX changes. The way `SavedObjectSaveModal` actually uses the `objectType`, it's more like an object label, description,  or name - it has no relationship to the saved object type (unless you happened to set it to the same value, which appears to have been the past behavior) and will work with any string. Likely out of scope for this PR, but I think we should rename `VIS_BUILDER_TYPE` to something a little less confusing. `VIS_BUILDER_TYPE_TITLE` maybe, based on the usage in the alias object?